app/gui2/tsconfig.json

```diff
@@ -10,5 +10,11 @@
     {
       "path": "./tsconfig.vitest.json"
     }
-  ]
+  ],
+  "compilerOptions": {
+    "skipLibCheck": true,
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+    "exactOptionalPropertyTypes": true
+  }
```

Comment:
This file should only contain references. If you need to cover more files of a new kind, make another dedicated tsconfig for them and reference it here.